docs/tempo/website/getting-started/_index.md

```diff
@@ -6,18 +6,31 @@ weight: 100
 # Getting started with Tempo
 
 Distributed tracing visualizes the lifecycle of a request as it passes through
-an application. There are a few components that must be configured in order to get a
+a set of applications. There are a few components that must be configured in order to get a
 working distributed tracing visualization.
 
+An example trace visualized in Grafana:
+
+<p align="center"><img src="../example-trace.png" alt="Example Trace"></p>
+
+This document discusses the four major pieces necessary to build out a tracing system: 
+Instrumentation, Tracing Pipeline, Tracing Backend and Visualization. If one were to build a diagram laying 
+out these pieces it may look something like this:
+
+<p align="center"><img src="../getting-started.png" alt="Tracing Overview"></p>
+
 ## 1. Client instrumentation
 
-#### OpenTelemetry Instrumentation SDKs
+#### Instrumentation SDKs
 
 The first building block to a functioning distributed tracing visualization pipeline
-is client instrumentation, which is the process of adding instrumentation points in the application that collects telemetry information. 
+is client instrumentation, which is the process of adding instrumentation points in the application that 
+creates and offloads spans. 
 
 SDKs that are available in the most commonly used programming languages are listed below -
 
+* [Jaeger](https://www.jaegertracing.io/docs/latest/client-libraries/)
+* [Zipkin](https://zipkin.io/pages/tracers_instrumentation)
 * [OpenTemeletry CPP](https://github.com/open-telemetry/opentelemetry-cpp)
 * [OpenTemeletry Java](https://github.com/open-telemetry/opentelemetry-java)
 * [OpenTemeletry JS](https://github.com/open-telemetry/opentelemetry-js)
```

Comment:
We list all the implementations of OpenTelemetry but we don't specify the languages supported by Jaeger nor Zipkin. Thoughts on making the OTEL stuff shorter? Probably we don't need that much detail... 

Another option is to list some of the languages supported by Jaeger and Zipkin.

Reply:
OTel doesn't have a very good page that lists all of their clients. This is the best I can find:

https://opentelemetry.io/docs/concepts/instrumenting/

It's ok, but honestly I feel like the individual repos are better.  I think we'll just link here and expect them to improve over time.